org.eclipse.corrosion/src/org/eclipse/corrosion/debug/DebugUtil.java

```diff
@@ -0,0 +1,43 @@
+/*********************************************************************
+ * Copyright (c) 2018 Red Hat Inc. and others.
```

Comment:
As far as  know, you're not working for Red Hat, so the copyright here is invalid.
I guess it should mention Fraunhofer (if you do it in the scope of your employment) or your name (if you do it on your own and your work contract allows you to keep IP on this).